bin/configurer/add-services.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -eu
+
+dir=$(dirname ${0})
+
+# Defaults are ok and create require ccd_gateway service in idam
+${dir}/utils/idam-create-service.sh ccd_gateway false ccd_gateway ccd_gateway_secret [\"http://localhost:3451/oauth2redirect\"]
```

Comment:
Can we move `false` inside the script and make it hardcoded there? It will be always false for us.

Reply:
Can we make redirect URL a single value? That would allow us have:

`${dir}/utils/idam-create-service.sh ccd_gateway false ccd_gateway ccd_gateway_secret http://localhost:3451/oauth2redirect`.

It is easier to read.

Reply:
Hardocded false.

Redirect URL single value, support only for single redirect,  I agree it is easier.